docs/new-audits.md

```diff
@@ -12,19 +12,26 @@ Lighthouse audits that surface in the report should be:
 - Measurable (especially for performance audits) or have clear pass/fail states. 
 - Not use 3rd party APIs for completing the audit check. 
 
-*Note: it's possible to submit a new audit proposal for something that you as a contributor feel very passionate about, but the Lighthouse team feels that it wouldn't add value to a significant portion of the Lighthouse users. In this case, we'd be open to having your audit be a part of the report when run with the full config on the Command Line.* 
 
 ## Process for creating a new audit
 1. Briefly scan the criteria we’ve laid out above. If you think the principles match with your proposed new audit, then proceed! 
 2. Next step is to create an issue on GitHub with answers to the following questions: 
-   - Description of audit and audit category (please include pass/fail states, and how it might be measured)
-   - Explanation of how it’s different from other audits
-   - What % of developers/pages will this impact (estimates OK, data points preferred) 
-   - How is the new audit making a better web for end users? (data points preferred)
-   - What is the resourcing situation (who will create the audits, maintain the audits, and write/maintain the documentation)
-   - Do you envision this audit in the Lighthouse report or the full config in the CLI? If in the report, which section? 
-   - How much support is needed from the Lighthouse team?
-   - Any other links or documentation that we should check out?
+```
+#### Provide a basic description of the audit
+#### How would the audit appear in the report? 
+<!-- How would the test look when passing? Would there be additional details available?
+     How would the test look when failing? What additional details are available? 
+     If the details are tabular, what are the columns?
+     If not obvious, how would passing/failing be defined? -->
+#### How is this audit different from existing ones?
+#### What % of developers/pages will this impact? 
```

Comment:
Yeah, the pro/con weighting for it is also different than it once was because now we have `notApplicable` and can just hide it, so visual noise is less of an issue (though still a concern in the limit). Now it's more maintenance/run time cost as the potential negative.